app/providers/yelp.py

```diff
@@ -5,11 +5,9 @@
 
 idPattern = re.compile("/([^/\?]*)(\?.*)?$")
 
-
 def resolve_with_key(key):
     return yelp3Client.request("/businesses/{0}".format(key))
 
-
```

Comment:
nit: the official python style standards (pep8) dictate 2 spaces between functions. We should decide on the style we want to follow.

(bonus: there's a command line tool for style checking: `pep8` & it's built into intellij by default)